org.goldcoinproject.goldcoin-qt.json

```diff
@@ -0,0 +1,71 @@
+{
+  "id": "org.goldcoinproject.goldcoin-qt",
+  "runtime": "org.freedesktop.Platform",
+  "sdk": "org.freedesktop.Sdk",
+  "runtime-version": "22.08",
+  "command": "goldcoin-qt",
+  "finish-args": [
+    /* Wayland and Xorg support */
+    "--socket=x11",
+    /* Sound */
+    "--socket=pulseaudio",
+    /* OpenGL access. */
+    "--device=dri",
+    /* IPC access. */
+    "--share=ipc",
+    /* Network acess. */
+    "--share=network"
+  ],
+  "modules": [
+    {
+      "name": "goldcoin",
+      "buildsystem": "simple",
+      "build-commands": [
+        "install -m 0744 -D -t /app/bin/internal bin/goldcoin-qt"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "only-arches": ["x86_64"],
+          "url": "https://github.com/goldcoin/goldcoin/releases/download/v0.14.9.1/goldcoin-0.14.9.1-x86_64-linux-gnu.tar.gz",
+          "sha256": "a9c6c6d6fdd9f021c1d8e22a18d4c83199f36362bf1579839f14af7c5c456204"
+        }
+      ]
+    },
+    {
+      "name": "misc",
+      "buildsystem": "simple",
+      "build-commands": [
+        "install -Dp -m 644 goldcoin-icon-transparent.svg /app/share/icons/hicolor/scalable/apps/org.goldcoinproject.goldcoin-qt.svg",
+        "install -Dp -m 644 goldcoin-qt.desktop /app/share/applications/org.goldcoinproject.goldcoin-qt.desktop",
+        "desktop-file-edit --set-key=Icon --set-value=org.goldcoin.goldcoin-qt.svg /app/share/applications/org.goldcoinproject.goldcoin-qt.desktop",
+        "install -Dp -m 644 org.goldcoinproject.goldcoin-qt.metainfo.xml /app/share/metainfo/org.goldcoinproject.goldcoin-qt.metainfo.xml",
+        "bash install_wrapper.sh"
+      ],
+      "sources": [
+        {
+          "type": "file",
+          "url": "https://raw.githubusercontent.com/expatjedi/flatpak-data/main/goldcoin-qt.desktop",
+          "sha256": "8b2a9c8da281be2f65161a2d8d49e6fcb4c7e9b78f78847b34cb3604c93b0384",
+          "commit":"b8e668d848c554977a76a35ecd0a3f654739c8b9"
+        },
+        {
+          "type": "file",
+          "path": "org.goldcoinproject.goldcoin-qt.metainfo.xml",
+          "commit":"b8e668d848c554977a76a35ecd0a3f654739c8b9"
+        },
+        {
+          "type": "file",
+          "url": "https://raw.githubusercontent.com/expatjedi/flatpak-data/main/goldcoin-icon-transparent.svg",
+          "sha256": "998d8756c777b6723ff2c78b885f7d48fe752c49d6224f858b65c5561b63c470",
+          "commit":"399d06fd616488917a6c3c5fa56def92c3e12696"
+        },
+        {
+          "type": "file",
+          "path": "install_wrapper.sh",
+          "commit":"b8e668d848c554977a76a35ecd0a3f654739c8b9"
```

Comment:
I'm pretty sure here that if flatpak-builder doesn't error it at least throw a warning since there is no commit for a source of type file.

This entry and the others.

Reply:
`(flatpak-builder:2): flatpak-builder-WARNING **: 16:41:48.764: Unknown property commit for type BuilderSourceFile`

Reply:
I checked many other flatpak packages and they don't have any commit after type:file line so i will keep it without commits.

Reply:
That's because as I pointed out `commit` isn't valid here.

But they don't point to `main` either at least when I review them (the one that point to a url from a git repo). Which is what I was requesting you to change. Because next time the file change it will break here.